com.stremio.Stremio.yml

```diff
@@ -57,7 +61,7 @@ modules:
   - install stremio-wrapper /app/bin/
 
   - mkdir -p /app/share/icons/hicolor/512x512/apps/
-  - cp squashfs-root/stremio.png /app/share/icons/hicolor/512x512/apps/com.stremio.Stremio.png
+  - cp com.stremio.Stremio.svg /app/share/icons/hicolor/512x512/apps/com.stremio.Stremio.svg
```

Comment:
Have you tried to build it from source? Last time I tried I needed Qt Web Engine, but it's now available here https://github.com/flathub/io.qt.qtwebengine.BaseApp. 
Of course, we can merge this as it's for now

Reply:
No i just wanted to get this initial build working and try to publish it, but i don't mind having another go! :)

Reply:
Once we merge this, i will work on building the source version